packages/web-app/hooks/useAgreement.tsx

```diff
@@ -77,6 +77,11 @@ export const useAgreementCreate = ({
 			}
 		},
 		onSuccess,
+		overrides: {
+			gasLimit: 190000,
+			// maxFeePerGas: 250000000,
+			// maxPriorityFeePerGas: 250000000,
+		},
```

Comment:
I have adjusted the gas limit for every operation. I was unable to reproduce the error that justified fixing gas fees neither with metamask or walletconnect. I can try again if you give me more context.

Reply:
if it works and each gas limit does enters into conflict, all good!